admin-src/utils/queries/index.js

```diff
@@ -2,44 +2,63 @@ import * as firebase from 'firebase'
 
 const STALE_STATUS_AFTER_DAYS = 10
 
-export const unansweredQuery = (onSnapshot) => {
-  return firebase
+export const unansweredQuery = onSnapshot => {
+  firebase
     .firestore()
     .collectionGroup('requests')
+    .orderBy('createdAt', 'asc')
     .where('amountOfAnswers', '==', 0)
     .onSnapshot(onSnapshot)
+
+  firebase
+    .firestore()
+    .collectionGroup('requests')
+    .orderBy('updatedAt')
+    .where('isLastCommentAdmin', '==', false)
+    .onSnapshot(onSnapshot)
 }
```

Comment:
@MPucheta is it OK to subscribe to two `onSnapshot` here? What about the removed `return` statement?

Reply:
Although I did not see this in execution, I guess it could work, kinda like executing two queries at the same time. Regardless if it works or not, there should be a `return` to give access to the `unsuscribe` function returned by `onSnapshot`.  A different approach is preferred